### ISSUE TITLE
Notes that are significant  need to be distinguished as normative?

### ISSUE THREAD

**Issue by spanchang** [w3c/wcag3#386]
WCAG 2.X section 5.1 states that notes are not normative. Is this how notes are to be treated in WCAG 3.0 too?
Consider In WCAG 3 (Sept 2025 draft): 
Note 1 for abbreviation: "This includes initialisms, acronyms, and numeronyms".
Likewise, for audio description:
Note For audiovisual media: "audio description provides information about actions, characters, scene changes, on-screen text, and other visual content". (This note actually clarifies what "important visual detail" in the main definition means without leaving it up for interpretation.
while:
Note 2: "Some companies have adopted what used to be an initialism as their company name. In these cases, the new name of the company is the letters (for example, Ecma) and the word is no longer considered an abbreviation".

Would it be possible to distinguish notes that are significant that they should be normative  and those that are not?
Is it possible to bifurcate these as, say, "Caution" or "Tips" and "Notes" and state that caution / tips are part of normative content? Then the first note  for abbreviation and the one for audio description above should perhaps be caution / tips and the one that relates to Ecma may remain a note. I have no problems if a different term is used to distinguish them instead of caution or tips as suggested if you think such a distinction is needed. I would not makes "notes" as the normative one as it will be a break from WCAG 2.X and lead to confusion.
Thanks

**Comment by GreggVan** [comment]
in any standard standard -- notes are always informative - not normative.  If we want WCAG3 to be picked up in other accessibility standards (like WCAG2 is now in EN 301 549) we need to continue to have notes be informative and not normative.

best

**Comment by spanchang** [comment]
True,notes are considered non-normative. This is explicit in 5.1 of WCAG 2.X.
However, this ticket #386  was prompted by a recent thread on the WAI-IG list:
https://lists.w3.org/Archives/Public/w3c-wai-ig/2025JulSep/0014.html
Captions obscuring text content of video - SC 1.2.2 

There appear to be two classes of notes: notes whose content appears to be advisory or illustrative, while there are those that appear to be more prescriptive. Or at least many regard them to be prescriptive and therefore sort of normative.
e.g. In WCAG 2.X For SC 1.4.5:  Logotypes (text that is part of a logo or brand name) are considered essential.
The Note#4 for captions (topic of WAI-IGG thread)  uses "should" in stating, "Captions should not obscure or obstruct relevant information in the video."
So these are meant to be advisory and best practice too.
The EN301 549 clearly states "it is a best practice ..." within some notes. should WCAG 3tag some notes in this fashion for clarity?
Thanks
